### ISSUE TITLE
What is the value of saving profile in a file?

### ISSUE THREAD

**Issue by SimonLab** [akitten/linkedin-public-profile-parser#27]
The html profile is saved on a file, however it doesn't seems that the program uses this file later on. Why do we need to write this file, can we get rid of it?
see [L37](https://github.com/akitten/linkedin-public-profile-parser/blob/df244ff9e986b6dee80a601235e709876cfec8b0/lib/fetcher.js#L37)

**Comment by nelsonic** [comment]
Yes, absolutely don't need html except if we want to add additional fields to the parser and want to re-scan the data without having to re-issue the requests.... Please feel free to PR. Thanks.
